crates/biome_js_formatter/src/js/expressions/arrow_function_expression.rs

```diff
@@ -473,6 +488,13 @@ impl Format<JsFormatContext> for ArrowChain {
             )
         });
 
+        let mut has_comment = false;
+        if let AnyJsFunctionBody::AnyJsExpression(AnyJsExpression::JsSequenceExpression(sequence)) =
+            &tail_body
+        {
+            has_comment = f.context().comments().has_comments(sequence.syntax());
+        }
```

Comment:
How about this? I think this suggestion needs formatting.
```suggestion
        let has_comment = matches!(
          &tail_body,
          AnyJsFunctionBody::AnyJsExpression(AnyJsExpression::JsSequenceExpression(sequence)) 
          if f.context().comments().has_comments(sequence.syntax())
        );
```